apps/api/src/app/events/services/mail-service/handlers/nodemailer.handler.ts

```diff
@@ -15,13 +15,23 @@ export class NodemailerHandler extends BaseHandler {
       secure: boolean;
       user: string;
       password: string;
+      dkim: {
+        domainName: string;
+        keySelector: string;
+        privateKey: string;
+      };
     } = {
       from,
       host: credentials.host,
       port: Number(credentials.port),
       secure: credentials.secure,
       user: credentials.user,
       password: credentials.password,
+      dkim: {
+        domainName: credentials.domain,
+        keySelector: credentials.accountSid,
+        privateKey: credentials.token,
+      },
```

Comment:
A small question, shouldn't it be either null or the object? Just wondering if nodemailer won't do problems if we specified this object and all it's values are bullish. Have you done some regression testing on this without saving the DKIM settings on the UI?

Reply:
Yes this is tested without DKIM settings and the email reaches my inbox 😄